docs/api/welcome/authentication.md

```diff
@@ -37,8 +37,23 @@ curl -X GET -u admin:admin -H 'Content-Type: application/json' https://127.0.0.1
 
 ### Bearer Authentication
 
-Bearer authentication (also called token authentication) is an HTTP authentication scheme that involves security tokens called bearer tokens. The bearer token is a cryptic API key, which can be generated by the server admin from the Settings UI or via a respective API call (<a href="https://docs.percona.com/percona-monitoring-and-management/details/api.html#generate-api-keys">read more about how to generate an API key</a>). The client must send the API key in the `Authorization` header when making requests to protected resources:
+Bearer authentication (also called token authentication) is an HTTP authentication scheme that involves security tokens called bearer tokens. The bearer token is a cryptic Service Token, or API key, which can be generated by the server admin from the Settings UI or via a respective API call (read more about how to generate <a href="https://docs.percona.com/percona-monitoring-and-management/details/api.html#generate-service-token">Service token</a> or <a href="https://docs.percona.com/percona-monitoring-and-management/details/api.html#generate-api-keys">API key</a>). The client must send the Service Token/API key in the `Authorization` header when making requests to protected resources:
 
+Service Token example:
+```shell
+curl -X GET -H 'Authorization: Bearer glsa_Fp0ggev31R58ueNJbJgYw7fIGfO3yKWH_746383ab' \
+  -H 'Content-Type: application/json' https://127.0.0.1/v1/version
+```
+
+You can use the Service Token in basic authentication as well:
+
+```shell
+curl -X GET -H 'Content-Type: application/json' \
+  https://service_token:glsa_Fp0ggev31R58ueNJbJgYw7fIGfO3yKWH_746383ab@127.0.0.1/v1/version
+```
+Service Token has usually prefix "glsa_". If you convert it from API key it will persist same. If you will use API key then it will be automatically converted into Service Account and Service Token.
```

Comment:
```suggestion
Service Token usually has "glsa_" prefix. If you convert it from API key, it will not change. If you use API key, then it will be automatically converted into Service Account and Service Token.
```